### PR TITLE
chore(import-export): fix number formatting for import label

### PR DESCRIPTION
**Pull request by gribnoysup**
|Before|After|
|---|---|
|image|image|

### CODE REVIEW

File: packages/compass-import-export/src/components/progress-bar/progress-bar.jsx [hunk 1]
```diff
@@ -133,13 +133,13 @@ class ProgressBar extends PureComponent {
 
     return (
       <div className={style('chart-wrapper')}>
-        {docsTotal && (
+        {!isNaN(docsTotal) && (
```
Comment (gribnoysup) [inline packages/compass-import-export/src/components/progress-bar/progress-bar.jsx:136]: Drive-by to fix `0` rendering for a split second (value needs to be false, null, or undefined for react not to render it)

**Review by addaleax — APPROVED** [review]
